models/user/badge.go

```diff
@@ -11,7 +11,8 @@ import (
 
 // Badge represents a user badge
 type Badge struct {
-	ID          int64 `xorm:"pk autoincr"`
+	ID          int64  `xorm:"pk autoincr"`
+	Slug        string `xorm:"UNIQUE"`
```

Comment:
Overall LGTM. The only question is I don't understand what `slug` means here.

(I am not a native English speaker, so if it is the right word, my question doesn't block)

Reply:
Is the length 255 enough?